vignettes/ebola.Rmd

```diff
@@ -53,10 +53,21 @@ library(tidybayes)
 library(modelr)
 library(patchwork)
 library(lubridate)
+library(cmdstanr)
 ```
 
 For users new to `epidist`, before reading this article we recommend beginning with the "[Getting started with `epidist`](http://epidist.epinowcast.org/articles/epidist.html)" vignette.
 
+# Using the cmdstanr backend
+
+As the models explored in this vignette are relatively complex, we recommend using the `cmdstanr` backend for fitting models as it is typically more performant than the default `rstan` backend.
+To use the `cmdstanr` backend, we first need to install CmdStan (see the README for more details).
```

Comment:
Link to the README?

Reply:
I wasn't sure which links would be the most portable so left for now